src/app/home/home.component.ts

```diff
@@ -125,7 +125,8 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
   logoutClick() {
     this.userService.endSessionLog().pipe(switchMap(() => {
       const obsArr = [ this.couchService.delete('_session', { withCredentials: true }) ];
-      if (this.userService.getConfig().name === this.userService.get().name) {

```

Comment:
this weird condition was added on https://github.com/open-learning-exchange/planet/pull/578/commits/43e83065d668b3e1a15d67e98c94e7cd359acae3
Yes we have been using same admin and config name in most cases but that will not always be the case. So we need to check config admin and user name